src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -29,7 +29,7 @@
 
     /**
      * The amount of buffer time, in milliseconds, to use when determining if an access token should be refreshed. For
-     * example, if REFRESH_EPSILON = 60000 and the access token expires in less than one minute, it will be refreshed.
+     * samples, if REFRESH_EPSILON = 60000 and the access token expires in less than one minute, it will be refreshed.
```

Comment:
Hmm I don't think these java doc comments should be modified, seems like `example` is still the right word for most of them